js/components/multi-input.less

```diff
@@ -0,0 +1,27 @@
+.multi-input {
+    margin-bottom: 20px;
```

Comment:
I would keep consistent in using `px` vs `rem` (`rem`-s are preferrable as for me)